app/src/androidTest/java/org/mozilla/fenix/perf/HomeFragmentBenchmark.kt

```diff
@@ -0,0 +1,150 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
chore (blocking): can we squash before landing? The individual commits are losing their story